packages/app/src/domain/topical/link-block/use-data-sitemap.tsx

```diff
@@ -0,0 +1,195 @@
+import siteText from '~/locale/index';
+import { LinkGroupProps } from './link-group';
+
+export function useDataSitemap(
+  base: 'landelijk' | 'veiligheidsregio' | 'gemeente',
+  code?: string,
+  data?: any
```

Comment:
This can be typed as

```suggestion
  data?: National | Regionaal | Municipal
```

I guess?

Reply:
🤩